.github/actions/build-sign-publish-chainlink/action.yml

```
@@ -9,13 +9,17 @@ inputs:
     default: "false"
     required: false
 
-  image-name:
-    description: The name of the image, should match the repository name in ECR
-    required: true
-
-  ecr-registry:
-    description: The ECR registry to push to, used in docker/login-action and for tagging images
-    default: public.ecr.aws/chainlink
+  ecr-prefix:
+    description: The ECR registry scope
+    default: public.ecr.aws
+    required: false
+  ecr-repo-path:
```

How do you feel about this?

```suggestion
  ecr-image-name:
```

I referenced https://docs.aws.amazon.com/AmazonECR/latest/userguide/docker-pull-ecr-image.html, where the image name format is `registry/repository`, private ECR registries use the default private ECR repository which has no `registry` component.

Then the description text could be 
```
The image name, in the format of [registry]/repository`, for private ECR repos, the registry name is optional, where for public ECR repos, it is required.

Eg. For a public ECR repo of `chainlink` and the public registry alias of `chainlinklabs`, we would have the image name of `chainlinklabs/chainlink`.

For a private ECR repo of `chainlink`, the image name would be `chainlink`
```

---

Ah that makes sense good reasoning, updated now but changed the description text a bit. Let me know what you think